src/common/CMakeLists.txt

```diff
@@ -7,12 +7,15 @@
 set(common_headers
```

Comment:
`common` is getting fairly large. Is it worth splitting this up a bit? I'm not sure how we would go about doing that though.

Reply:
It seems like there may be a reasonable split between "utility" stuff - loggers, initialization, termination, and FEM-related stuff - boundary conditions, mesh stuff, etc.

Reply:
I'll add an issue on this